tests/Entity/JoindInUserTest.php

```diff
@@ -40,6 +43,17 @@ public function testSetDisplayName()
         $this->markTestSkipped('Skipping');
     }
 
+    public function testPromoteToOrganizer()
+    {
+        $this->joindInUser->promoteToOrganizer();
+        self::assertEquals(true, $this->joindInUser->isOrganizer());
+    }
+
+    public function testisOrganizer()
```

Comment:
Can we rename this to 'testIsOrganizer' .. pleasseeeeee ;)

Reply:
sorry, totally missed that one!  :blush: